Assets/Plugins/Source/Core/Utility/JsonUtility.cs

```diff
@@ -22,21 +22,112 @@
 
 namespace PlayEveryWare.EpicOnlineServices.Utility
 {
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
```

Comment:
I am kind of unsure about these extra dependencies. IIs there a way to work around this?

Reply:
The Newtonsoft stuff is already included in the project, so no new dependencies should be added.